PWGGA/GammaConv/macros/AddTask_GammaCaloIso_pp.C

```diff
@@ -20,113 +20,94 @@
 //pp together with all supporting classes
```

Comment:
Shouldn't this addtask be called AddTask_GammaConvCaloIso_pp.C? At least you are using PCM-Calo reco and not just Calo. Furthermore you should have a look some of your couts have copy paste errors.